### ISSUE TITLE
CTD when trying to start a game

### ISSUE THREAD

**Issue by ikshu-n** [LoneGazebo/Community-Patch-DLL#11278]
Current Version: 4.16.0_
with EUI

List of Other Mods_
Ultimately, I will use 4UC and Even More Luxuries, but this crash occurs even without those mods, just with VP+EUI.

![mods](https://github.com/user-attachments/assets/57e850db-3a05-4fb1-9e8e-c9dae7a0e9b7)

Describe the Issue_
When I click "Next" the in the mod selection screen, a loading screen appears for a few seconds. Then, the game crashes to the desktop.

I imagine the minidump would have some helpful information, but I cannot seem to open it in Visual Studio (this is my first time trying to open a dmp file, so I might be doing something wrong).

CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_
[CvMiniDump.dmp](https://github.com/user-attachments/files/17011642/CvMiniDump.dmp)

Here is my Logs folder:
[Logs.zip](https://github.com/user-attachments/files/17011649/Logs.zip)

**Comment by azum4roll** [comment]
Which mods did you activate? Which menu did you use to start the game? Do you have any mods subscribed on the Steam Workshop?

**Comment by ikshu-n** [comment]
When trying to replicate the crash, the nature of the crash changed. Previously, I had been able to make it past the mod selection screen to the game setup screen. Now, the game is crashing when I click "next" in the mod selection screen. I have updated my original post accordingly.

I have also updated the post with the information you requested:
I only have the VP+EUI mod active. I do not have any mods subscribed in the Steam workshop. I attached my logs folder to the original post.

**Comment by Hokath** [comment]
When this inexplicably happens to me, clearing the Cache folder usually fixes it.

**Comment by ikshu-n** [comment]
Unfortunately, clearing the cache did not help.

**Comment by azum4roll** [comment]
Something looks messed up in your game files. Try verifying file integrity on Steam.

**Comment by ikshu-n** [comment]
I verified the file integrity on Steam, but the crash still happens. However, when I verify the file integrity, it says that some number of files failed to validate. But it has always given this message, everytime I have validated files in the past.

**Comment by ikshu-n** [comment]
I reinstalled the game, but it still crashes.

**Comment by ikshu-n** [comment]
I was starting in the Ancient era

**Comment by azum4roll** [comment]
Can you post the new logs?

**Comment by ikshu-n** [comment]
Here are the new logs. This is after reinstalling the game and the mods. It still crashes at the same point.
[Logs.zip](https://github.com/user-attachments/files/17155754/Logs.zip)

**Comment by azum4roll** [comment]
Database.log still shows that you have possibly duplicated game files, which makes everything loaded twice.

```You can enable it by setting 'ValidateGameDatabase = 1' in config.ini```

Can you enable this?

**Comment by ikshu-n** [comment]
The game works fine without mods. I will try enabling what you suggested.

**Comment by ikshu-n** [comment]
I set 'ValidateGameDatabase = 1' in config.ini but the game still crashes at the same point. Was there something else I needed to do after editing the config file?

**Comment by azum4roll** [comment]
It just enables database validation logs. Now please upload your logs again.

**Comment by ikshu-n** [comment]
Here are the new logs
[Logs.zip](https://github.com/user-attachments/files/17175668/Logs.zip)

**Comment by axatin** [comment]
Could it be that you still have a modpack installed? (in the DLC folder)

**Comment by ikshu-n** [comment]
No, I deleted the modpack files beforehand.

**Comment by ikshu-n** [comment]
I seem to have "solved" this issue.

I deleted everything in my MODS folder, cleared the cache, and removed the modpack files from my DLC folder. I started a game to make sure that it was working without mods.

I had tried all of this earlier when someone asked if my game works without mods. Back then, I said yes because the game started fine after getting rid of all mod files. However, what I did not notice was that the game was still modded (I've basically forgotten what unmodded civ looks like)!

So now, the game works. It is using some version of the modpack (EUI+4UC+even more resources). I cannot see the files.

Now my main problem is how to update the mods/get rid of the old version.

Here is a picture of my DLC folder. Where can I find the old modpack files to delete them and install a new version?
![DLC](https://github.com/user-attachments/assets/5f066973-490e-42fc-a3e8-c409b53d8e56)
